diff -Nru gfontview-0.4.1/configure.in gfontview-0.4.1.new/configure.in
--- gfontview-0.4.1/configure.in	Sun May 28 21:16:11 2000
+++ gfontview-0.4.1.new/configure.in	Sat Jul  1 20:59:45 2000
@@ -30,15 +30,6 @@
   AC_DEFINE_UNQUOTED(DEFAULT_FONTDIR, "$withval"),
   AC_DEFINE_UNQUOTED(DEFAULT_FONTDIR, "/usr/share/ghostscript/fonts")
   )
-AC_ARG_ENABLE(debug, 
-  [  --enable-debug[=yes/no] turn on debugging [no]],,
-  )
-if test "$enable_debug" = "yes"; then
-  CXXFLAGS="-Wall -Wno-unused -g"
-else
-  CXXFLAGS="-O2"
-fi
-
 
 dnl ---------------------------------------------------------------------------
 dnl i18n
