diff -Nru gfontview-0.3.2/configure.in gfontview-0.3.2.new/configure.in
--- gfontview-0.3.2/configure.in	Fri Dec 10 22:55:36 1999
+++ gfontview-0.3.2.new/configure.in	Wed Dec 22 16:28:43 1999
@@ -8,20 +8,6 @@
 GFV_MICRO_VERSION=2
 GFV_VERSION=$GFV_MAJOR_VERSION.$GFV_MINOR_VERSION.$GFV_MICRO_VERSION
 
-dnl ---------------------------------------------------------------------------
-dnl for automake
-dnl ---------------------------------------------------------------------------
-AUTOMAKE_OPTIONS=no-dependencies
-VERSION=$GFV_VERSION
-PACKAGE=gfontview
-AC_ARG_ENABLE(debug, 
-  [  --enable-debug=[no/yes] turn on debugging [default=no]],,
-  )
-if test "$enable_debug" = "yes"; then
-  CXXFLAGS="-Wall -Wno-unused -g"
-else
-  CXXFLAGS="-O2"
-fi
 
 dnl ---------------------------------------------------------------------------
 dnl Initialize automake now
